src/views/LobbyView.vue

```diff
@@ -13,6 +13,17 @@
         </h1>
       </v-col>
     </v-row>
+    <v-row>
+      <v-col offset="5">
+        <v-switch
+          v-model="gameStore.isRanked"
+          :label="gameStore.isRanked ? 'Ranked' : 'Normal'"
```

Comment:
```suggestion
          :label="gameStore.isRanked ? 'Ranked' : 'Casual'"
```

Reply:
Now that we're translating this page, we should generate the label using `i18n()`

Reply:
```suggestion
          :label="gameStore.isRanked ? t('lobby.ranked') : t('lobby.casual')"
```